### PR TITLE
Deb deploy script

### PR DESCRIPTION
**Pull request by gijzelaerr**

### CODE REVIEW

**Comment by ghost** [comment]
Does it work as-is?

**Comment by ghost** [comment]
default debian install does not have `curl` installed.

**Comment by ghost** [comment]
```
# update the IPv4 CIDR and IPv6 prefix to random IP ranges and set the extIf
vpn-server-api-update-ip --profile internet --host ${VPN_FQDN} --ext ${EXTERNAL_IF}
PHP Warning:  require_once(/usr/share/php/Symfony/Component/Translation/autoload.php): failed to open stream: No such file or directory in /usr/share/php/Twig/Extensions/autoload.php on line 44
PHP Fatal error:  require_once(): Failed opening required '/usr/share/php/Symfony/Component/Translation/autoload.php' (include_path='.:/usr/share/php') in /usr/share/php/Twig/Extensions/autoload.php on line 44
```

**Comment by gijzelaerr** [comment]
well that last one is weird, for me it works fine. Maybe I forgot to upload the package. What version of `vpn-server-api`? Could be the the auto dependency walking failed in this case. Do you have `php-symfony-translation` installed? If not `php-twig-extensions` has a missing dependency on `php-symfony-translation`. Can you verify if it helps if you install `php-symfony-translation`?

**Comment by ghost** [comment]
```ii  vpn-server-api 0.1-1        amd64        VPN Server API```

`php-symfony-translation` was not installed. I installed it now! 

```
root@debvpn:/home/debian# vpn-server-api-update-ip --profile internet --host debvpn.tuxed.net --ext eth0
IPv4 CIDR  : 10.119.217.0/24
IPv6 prefix: fd6a:758c:c4e3:4290::/60
root@debvpn:/home/debian# 

```

**Comment by ghost** [comment]
Had to at least do this as well:

```
a2ensite debvpn.tuxed.net
a2enmod headers
a2enmod rewrite
a2enmod proxy
```

Also had to change the log folder in the debvpn.tuxed.net.conf.

**Comment by gijzelaerr** [comment]
https://github.com/eduvpn-debian/packaging/issues/4

**Comment by ghost** [comment]
a2enconf vpn-server-api
a2enconf vpn-user-portal
a2enconf vpn-admin-portal

Also, on Debian the PHP socket is `/run/php/php7.0-fpm.sock`.

**Comment by ghost** [comment]
```
==> debvpn.tuxed.net_ssl_error_log <==
[Mon Jul 17 09:18:18.343762 2017] [proxy:warn] [pid 27995:tid 140506923583232] [client 2001:610:450:10::1000:47234] AH01144: No protocol handler was valid for the URL /vpn-user-portal/index.php. If you are using a DSO version of mod_proxy, make sure the proxy submodules are included in the configuration using LoadModule.

```

**Comment by ghost** [comment]
Ah, the trick is `a2enmod proxy_fcgi`

**Comment by ghost** [comment]
It seems to *almost* work, the login works! But still an issue with templates it seems...

```
==> debvpn.tuxed.net_ssl_error_log <==
[Mon Jul 17 09:21:37.773064 2017] [proxy_fcgi:error] [pid 28414:tid 140514074486528] [client 2001:610:450:10::1000:47292] AH01071: Got error 'PHP message: PHP Fatal error:  Uncaught Error: Call to undefined method Twig_Node_Expression_Name::getTemplateLine() in /usr/share/php/Twig/Extensions/Node/Trans.php:143\nStack trace:\n#0 /usr/share/php/Twig/Extensions/Node/Trans.php(42): Twig_Extensions_Node_Trans->compileString(Object(Twig_Node))\n#1 /usr/share/php/Twig/Node.php(110): Twig_Extensions_Node_Trans->compile(Object(Twig_Compiler))\n#2 /usr/share/php/Twig/Node.php(110): Twig_Node->compile(Object(Twig_Compiler))\n#3 /usr/share/php/Twig/Compiler.php(97): Twig_Node->compile(Object(Twig_Compiler))\n#4 /usr/share/php/Twig/Node/Module.php(277): Twig_Compiler->subcompile(Object(Twig_Node_Body))\n#5 /usr/share/php/Twig/Node/Module.php(80): Twig_Node_Module->compileDisplay(Object(Twig_Compiler))\n#6 /usr/share/php/Twig/Node/Module.php(52): Twig_Node_Module->compileTemplate(Object(Twig_Compiler))\n#7 /usr/share/php/Twig/Compiler.php(86): Twig_Node_Module->compile(Object(Twig_Compiler))\n#8 /usr/share/php/Twig/Environment.php(651): Twig_Compiler->compile(Object(T...\n', referer: https://debvpn.tuxed.net/vpn-admin-portal/stats
```

**Comment by ghost** [comment]
Firewall: https://github.com/eduvpn/documentation/blob/e83d7ea8e2605fad5b4cb81388278a9afd630ba6/attic/deploy_debian_node.sh#L190

**Comment by ghost** [comment]
the error from https://github.com/eduvpn/documentation/pull/16#issuecomment-315705452 is still there.

**Comment by gijzelaerr** [comment]
i don't know what it means? API version mismatch?

**Comment by ghost** [comment]
Okay, let me rephrase that: do the packages actually work for you on Debian 9?

**Comment by gijzelaerr** [comment]
they install and basic functionality seemed to be working, but I thought we agreed that you would check if everything is working since probably it is much easier for you to set up. I don't know what that error means. is there a autoload statement missing somewhere?

**Comment by ghost** [comment]
Created one based on deploy.sh and that works for the most part.
